hollow/src/main/java/com/netflix/hollow/api/client/HollowClientUpdater.java

```diff
@@ -51,6 +53,46 @@
 
     private HollowFilterConfig filter;
 
+    /**
+     *
+     * @param transitionCreator
+     * @param updateListeners
+     * @param apiFactory
+     * @param doubleSnapshotConfig
+     * @param hashCodeFinder
+     * @param objectLongevityConfig
+     * @param objectLongevityDetector
+     * @param metrics
+     * @param metricsCollector
+     *
+     * @deprecated use {@link #HollowClientUpdater(HollowConsumer.BlobRetriever, List, HollowAPIFactory,
+     *                                             HollowConsumer.DoubleSnapshotConfig, HollowObjectHashCodeFinder,
+     *                                             HollowConsumer.ObjectLongevityConfig,
+     *                                             HollowConsumer.ObjectLongevityDetector, HollowConsumerMetrics,
+     *                                             HollowMetricsCollector, CompletableFuture)} instead
+     */
+    @Deprecated
```

Comment:
Since HollowClientUpdater is internal is there an explicit need to mark as deprecated, instead can the constructor just be removed? This leads to a wider point: is the new constructor needed? HollowClientUpdater could control the instantiation of the CF rather than that being "percolated" up to  HollowConsumer that creates it then passed ownership to HollowClientUpdater.